src/io/flutter/editor/NativeEditorNotificationProvider.java

```diff
@@ -17,13 +17,30 @@
 import com.intellij.ui.EditorNotifications;
 import com.intellij.ui.HyperlinkLabel;
 import icons.FlutterIcons;
-import org.jetbrains.annotations.NonNls;
+import io.flutter.actions.ActionWithAnalytics;
+import io.flutter.actions.OpenInXcodeAction;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 public class NativeEditorNotificationProvider extends EditorNotifications.Provider<EditorNotificationPanel> implements DumbAware {
   private static final Key<EditorNotificationPanel> KEY = Key.create("flutter.native.editor.notification");
 
+  private static final ActionWithAnalytics OPEN_IN_XCODE_ACTION = new OpenInXcodeAction() {
+    @NotNull
+    @Override
+    public String getAnalyticsId() {
+      return "openInXcodeFromBanner";
```

Comment:
We don't have an existing pattern for this; perhaps `OpenInXcode.banner`?

Reply:
👍